R/modeltests.R

```diff
@@ -300,6 +300,12 @@ evaluateRuns <- function(model, mydir, gitPath, compScen, email, mattermostToken
     if (grsi[, "Conv"] %in% c("converged", "converged (had INFES)")) {
       setwd(i)
       message("Changed to ", normalizePath("."))
+      # Use the fulldata.gdx of a successful SSP2EU-NPi-AMT to update the gdx on the RSE server that is used for testing convGDX2MIF
+      if (grepl("SSP2EU-NPi-AMT", rownames(grsi))) {
+        gdxOnRseServer <- "rse:/webservice/data/example/remind2_test-convGDX2MIF_fulldata.gdx"
```

Comment:
Sorry for being late to the party, but it seems you overwriting the file [used for remind2 testing](https://github.com/pik-piam/remind2/blob/master/tests/testthat/test-convGDX2mif.R#L22), while in [this discussion](https://github.com/pik-piam/remind2/issues/524), there seems to be quite some support for keeping a file as well from the last release, so maybe we should push that to a different filename?

Reply:
Oh, thanks for notifying.